workloads/hyper-scale/common.sh

```diff
@@ -2,12 +2,13 @@
 set -x
 
 source env.sh
+TEMP_DIR=`mktemp -d`
 
 prep(){
     if [[ -z $(go version) ]]; then
         curl -L https://go.dev/dl/go1.18.2.linux-amd64.tar.gz -o go1.18.2.linux-amd64.tar.gz
-        tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz
-        export PATH=$PATH:/usr/local/go/bin
+        tar -C ${TEMP_DIR}/ -xzf go1.18.2.linux-amd64.tar.gz
+        export PATH=${TEMP_DIR}/go/bin/:$PATH
     fi
     if [[ ${HYPERSHIFT_CLI_INSTALL} != "false" ]]; then
         echo "Remove current Hypershift CLI directory.."
```

Comment:
@krishvoor why aren't we doing this for hypershift binaries ?

Reply:
@mukrishn here's the one, where the changes are introduced
https://github.com/cloud-bulldozer/e2e-benchmarking/pull/463/files